### PR TITLE
Experiment: streaming job worker should be resilient to full cluster restart (brokers)

### LINKED ISSUE #444: Hypothesis: streaming job worker should be resilient to full cluster restart (brokers)
### Hypothesis

A streaming job worker should still complete all started process instances even if all the brokers are restarted.

### PR DESCRIPTION
**Pull request by npepinpe**
**Description**

Adds a new experiment which ensures we can still create and complete instances with streaming job workers even after restarting all brokers.

Tested manually using the integration test.

closes #444

### CODE REVIEW

**Comment by npepinpe** [comment]
Depends on #453, so please review that first.

File: go-chaos/internal/chaos-experiments/camunda-cloud/job-push/cluster-restart.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
+{
+  "version": "0.1.0",
+  "title": "Job push multiple leader change experiment",
```
Comment (Zelldon) [inline go-chaos/internal/chaos-experiments/camunda-cloud/job-push/cluster-restart.json:3]:
```suggestion
  "title": "Job push cluster restart experiment",
```

File: go-chaos/internal/chaos-experiments/camunda-cloud/job-push/cluster-restart.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  },
+  "method": [
+    {
+        "name": "Restart all brokers",
+        "type": "probe",
+        "tolerance": 0,
+        "provider": {
+            "type": "process",
+            "path": "zbchaos",
+            "arguments": ["restart", "broker", "--all"],
```
Comment (Zelldon) [inline go-chaos/internal/chaos-experiments/camunda-cloud/job-push/cluster-restart.json:70]: ❓  What about terminate

**Review by Zelldon — APPROVED** [review]
👍🏼

**Review by Zelldon — APPROVED** [review]

**Pull request merged**